### PR TITLE
reorganizing the repo

### PR DESCRIPTION
**Pull request by mfaghihi**
doesn't compile yet. needs to add scripts and fix errors

### CODE REVIEW

**Review by drewstone — CHANGES_REQUESTED** [review]
Build and CI is failing.

File: arkworks-benchmarks/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    "poseidon_bn254_x17_3",
+    "poseidon_bn254_x3_3",
+    "poseidon_bn254_x3_5",
+    "poseidon_circom_bn254_x5_3",
+    "poseidon_circom_bn254_x5_5",
+]
+
+test = ["poseidon_bn254_x5_5", "poseidon_bn254_x5_3", "poseidon_circom_bn254_x5_3"]
+r1cs = []
+std = ["ark-std/std"]
```
Comment (filiplazovic) [inline arkworks-benchmarks/Cargo.toml:68]: We don't need these features in benchmarks

File: arkworks-benchmarks/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+poseidon_circom_bn254_x5_3 = []
+poseidon_circom_bn254_x5_5 = []
+
+mimc_220_ed_on_bn254 = []
+
+[[bench]]
+name = "bridge_bench"
+path = "benches/bench.rs"
+harness = false
+required-features = ["r1cs", "std", "default_poseidon", "test"]
```
Comment (filiplazovic) [inline arkworks-benchmarks/Cargo.toml:112]: We may not need this. Lets make it a binary, for simplicity

File: arkworks-benchmarks/Cargo.toml (hunk 1 above)
Comment (filiplazovic) [reply arkworks-benchmarks/Cargo.toml:68]: And check other crates too

Comment (filiplazovic) [reply arkworks-benchmarks/Cargo.toml]: Please check other crates. Only `arkworks-utils` need these features

**Review by filiplazovic — APPROVED** [review]

**Pull request merged**